aea/cli/issue_certificates.py

```diff
@@ -60,7 +67,7 @@ def _process_certificate(ctx: Context, cert_request: CertRequest):
         )
         if connection_private_key_path is None:
             raise ClickException(
-                f"Cannot find connection private key with id '{key_identifier}'"
+                f"Cannot find connection private key with id '{key_identifier}'. Connection '{connection_id}' requires this. Please use `aea generate-key {key_identifier} connection_{key_identifier}_private_key.txt` and `aea add-key {key_identifier} connection_{key_identifier}_private_key.txt --connection` to add a connection private key with id '{key_identifier}'."
```

Comment:
Maybe add new lines characters `\n` to break long error message?
Also, for better readability, useful to break long strings in multiple lines (but it is orthogonal to the above sentence).